mne/report.py

```diff
@@ -839,40 +861,25 @@ def add_images_to_section(self, fnames, captions, scale=None,
                                              img_klass=img_klass,
                                              caption=caption,
                                              width=scale,
+                                             comments=comments,   
                                              show=True)
             self.fnames.append('%s-#-%s-#-custom' % (caption, sectionvar))
             self._sectionlabels.append(sectionvar)
             self.html.append(html)
 
-    def add_htmls_to_section(self, htmls, captions, section='custom'):
-        """Append htmls to the report.
-
-        Parameters
-        ----------
-        htmls : str | list of str
-            An html str or a list of html str.
-        captions : str | list of str
-            A caption or a list of captions to the htmls.
-        section : str
-            Name of the section. If section already exists, the images
-            will be appended to the end of the section.
-        """
-        htmls, captions = self._validate_input(htmls, captions, section)
-        for html, caption in zip(htmls, captions):
-            caption = 'custom plot' if caption == '' else caption
-            sectionvar = self._sectionvars[section]
-
-            self.fnames.append('%s-#-%s-#-custom' % (caption, sectionvar))
-            self._sectionlabels.append(sectionvar)
-            self.html.append(html)
-
-    def render_bem(self, subject, decim=2, n_jobs=1, subjects_dir=None):
```

Comment:
But wait, we just want a simple function to inject HTML code into the report, why remove it?